Makefile

```diff
@@ -22,3 +22,9 @@ update: update-deps init
 ui:
 	cd ui && npm run lint:fix
 	cd ui && node_modules/.bin/gatsby build --prefix-paths
+
+# Filter out errors from modules that do not have Sphinx-compatible
+# documentation.
+.PHONY: docs
+docs:
+	tox -e docs | egrep -v ' (fastapi|httpx|pydantic|starlette)\.'
```

Comment:
So this is because you're using single-backtick syntax with API names that don't use intersphinx? I guess in principle you could use double backticks to show that you're not trying to make a link to those APIs.

Reply:
I was sort of hopeful that maybe Sphinx inventories would show up in the future, and the output looks a bit better than literal text, I think, although I don't feel strongly about it.